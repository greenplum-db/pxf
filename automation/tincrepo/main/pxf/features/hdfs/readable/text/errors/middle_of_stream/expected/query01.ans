-- @description query01 for PXF HDFS Readable wrong type

-- start_matchsubs
--
-- m/(ERROR|WARNING):.*'\d+\.\d+\.\d+\.\d+:\d+'.*/
-- s/'\d+\.\d+\.\d+\.\d+:\d+'/'SOME_IP:SOME_PORT'/
--
-- m/DETAIL/
-- s/DETAIL/CONTEXT/
--
-- m/CONTEXT:  External table error_on_10000.*/
-- s/CONTEXT:  External table error_on_10000.*/CONTEXT:  External table error_on_10000/
--
-- end_matchsubs

SELECT * FROM error_on_10000 ORDER BY num ASC;
ERROR:  transfer error (18): Transferred a partial file from 'SOME_IP:SOME_PORT' (libchurl.c:xxx)
DETAIL:  External table error_on_10000
