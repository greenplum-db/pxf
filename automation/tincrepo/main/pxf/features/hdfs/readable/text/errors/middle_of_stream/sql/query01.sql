-- @description query01 for PXF HDFS Readable error in the middle of stream

-- start_matchsubs
--
-- m/(ERROR|WARNING):.*'\d+\.\d+\.\d+\.\d+:\d+'.*/
-- s/'\d+\.\d+\.\d+\.\d+:\d+'/'SOME_IP:SOME_PORT'/
--
-- m/CONTEXT:  \n/
-- s/CONTEXT:  \n/CONTEXT:  /
--
-- m/External table error_on_10000/
-- s/.*External table error_on_10000.*/CONTEXT:  External table error_on_10000/
--
-- end_matchsubs

SELECT * FROM error_on_10000 ORDER BY num ASC;
