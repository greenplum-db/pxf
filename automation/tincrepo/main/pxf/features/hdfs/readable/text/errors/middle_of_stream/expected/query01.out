-- @description query01 for PXF HDFS Readable error in the middle of stream
-- start_matchsubs
--
-- m/(ERROR|WARNING):.*'\d+\.\d+\.\d+\.\d+:\d+'.*/
-- s/'\d+\.\d+\.\d+\.\d+:\d+'/'SOME_IP:SOME_PORT'/
--
-- m/CONTEXT:  \n/
-- s/CONTEXT:  \n/CONTEXT:  /
--
-- m/External table error_on_10000/
-- s/.*External table error_on_10000.*/CONTEXT:  External table error_on_10000/
--
-- m/libchurl\.c/
-- s/libchurl\.c:\d+/libchurl\.c:xxx/
  
-- end_matchsubs
SELECT * FROM error_on_10000 ORDER BY num ASC;
ERROR:  transfer error (18): Transferred a partial file from 'SOME_IP:SOME_PORT' (libchurl.c:xxx)
DETAIL:  curl error buffer: transfer closed with outstanding read data remaining
CONTEXT:  External table error_on_10000, line 10001 of file pxf://pxf_automation_data/data?FRAGMENTER=org.greenplum.pxf.plugins.hdfs.HdfsDataFragmenter&ACCESSOR=org.greenplum.pxf.automation.testplugin.ThrowOn10000Accessor&RESOLVER=org.greenplum.pxf.plugins.hdfs.StringPassResolver
